components/list/list-item-generic-layout.js

```diff
@@ -69,7 +69,10 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 					[control-end content-start] minmax(0, auto)
 					[content-end actions-start] minmax(0, min-content)
 					[end actions-end];
-				grid-template-rows: [main-start] [main-end nested-start] [nested-end];
+				grid-template-rows:
+					[main] minmax(0, min-content)
+					[add-start add-end] minmax(0, min-content)
+					[nested-start nested-end] minmax(0, min-content);
```

Comment:
So we're adding a new grid-line, named by both `add-start` and `add-end`. It's a little peculiar that a gridline is named as both the start and end line, with nothing between them. I'm not 100% certain the new grid-lines are doing what we intend, but maybe we don't notice because the content is so short.